subsys/bootloader/Kconfig

```diff
@@ -165,9 +154,8 @@ config SB_BPROT_IN_DEBUG
 endif # SECURE_BOOT_DEBUG
 endif # IS_SECURE_BOOTLOADER
 
-if IS_SECURE_BOOTLOADER || SECURE_BOOT || SECURE_BOOT_CRYPTO_CLIENT
```

Comment:
Is this correct has dependencies be changed?

Reply:
I think I saw changes that allowed this to be the case but I'm not sure so just an answer her would suffice.

Reply:
Yes, it's correct. The _CRYPTO_CLIENT config is gone completely, SECURE_BOOT_CRYPTO is used instead. But the include of bl_crypto is gone from inside this `if` anyways :).